game-core/src/main/java/games/strategy/engine/framework/startup/mc/ServerModel.java

```diff
@@ -112,13 +112,6 @@ static RemoteName getObserverWaitingToStartName(final INode node) {
   private CountDownLatch removeConnectionsLatch = null;
   private final Observer gameSelectorObserver = (observable, value) -> gameDataChanged();
 
-  public ServerModel(final ServerConnectionProps props) {
-    this(GameRunner.getGameSelectorModel(),
-        GameRunner.getSetupPanelModel(),
-        ServerModel.InteractionMode.SWING_CLIENT_UI);
-    GameRunner.getSetupPanelModel().setServerMode(this, props);
```

Comment:
Don't know if you want to keep pulling this thread, but it looks like this was the only caller of `SetupPanelModel#setServerMode()`.

Also, `GameRunner#getGameSelectorModel()` and `GameRunner#getSetupPanelModel()` seem to only be used by this constructor.

Reply:
Good catch, updated